### DISCUSSION TITLE
Publish packages on GitHub releases

### DISCUSSION THREAD

**Discussion by codenyte** [modrinth/.github#259] [Feature requests]
Publish the binary packages that are available on the website (https://modrinth.com/app) on GitHub releases in the [theseus](https://github.com/modrinth/theseus) repository.

**Comment by blryface** [comment]
It.. is?

**Comment by blryface** [comment]
The hell, it isnt

**Comment by blryface** [comment]
I swear I've seen the buttons link to githubusercontent

**Comment by Minenash** [comment]
Maybe from the action artifacts?
https://github.com/modrinth/code/actions/runs/10609703254
